plugins/package-managers/gleam/src/main/kotlin/GleamManifest.kt

```diff
@@ -29,10 +29,10 @@ import kotlinx.serialization.Serializable
 @Serializable
 internal data class GleamManifest(
     /** List of all resolved packages including transitive dependencies. */
-    val packages: List<Package>,
+    val packages: List<Package> = emptyList(),
```

Comment:
> to allow easier construction of empty Manifests.

Hmm, I'm not sure about that rationale. We usually introduce `EMPTY` constant for that purpose (see e.g. [this](https://github.com/oss-review-toolkit/ort/blob/60316b797569f05fc7835f6e2f3b2bb72340f71b/model/src/main/kotlin/Identifier.kt#L59)), and adding default values for serializable classes might have the unwanted side effect that (invalid?) files that lack these properties are still read properly.

So if you actually do expect these properties to be present in the input data, I believe we should not use default values here.